integration_tests/utils.py

```diff
@@ -445,6 +439,49 @@ async def deploy_contract_async(
     return w3.eth.contract(address=address, abi=abi)
 
 
+def build_contract(name):
```

Comment:
can we have a function with signature of `def build_contract(source: str) -> ArtifactDict`